sds_data_manager/lambda_code/SDSCode/dependency_config.csv

```diff
@@ -111,6 +111,7 @@ swapi, l1, sci, swapi, l2, sci, HARD, DOWNSTREAM
 
 swapi, l2, sci, swapi, l3a, alpha-sw, HARD, DOWNSTREAM
 swapi, l2, sci, swapi, l3a, proton-sw, HARD, DOWNSTREAM
+swapi, l2, sci, swapi, l3b, combined, HARD, DOWNSTREAM
```

Comment:
Is SWAPI l2 data still an input to l3b data product or are l3a products input to l3b data product?

Reply:
Yes, the l3b data product is calculated directly from the l2 data and does not depend on l3a.
- Ted and Peter

Reply:
cool. I wanted to check this before approving.